src/vm/stubgen.h

```diff
@@ -273,6 +273,15 @@ class TokenLookupMap
         size_t size = pSrc->m_qbEntries.Size();
         m_qbEntries.AllocThrows(size);
         memcpy(m_qbEntries.Ptr(), pSrc->m_qbEntries.Ptr(), size);
+
+        m_signatures.Preallocate(pSrc->m_signatures.GetCount());
+        for (COUNT_T i = 0; i < pSrc->m_signatures.GetCount(); i++)
+        {
+            const CQuickBytesSpecifySize<16>& src = pSrc->m_signatures[i];
+            CQuickBytesSpecifySize<16>& dst = *m_signatures.Append();
+            dst.AllocThrows(src.Size());
+            memcpy(dst.Ptr(), src.Ptr(), src.Size());
+        }
```

Comment:
This is used from here: https://github.com/dotnet/coreclr/blob/d8d6d8a53217806a0f3866e3156e10e1d1b06084/src/vm/ilstubresolver.cpp#L464-L477